bin/run_pulse_ingestion

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SRC_DIR=$(dirname "$(dirname "${BASH_SOURCE[0]}")")
+cd $SRC_DIR
+
+PROJECT_ROOT=$(readlink -f ../)
+PATH=$PROJECT_ROOT/venv/bin:$PATH
+
+source /etc/profile.d/treeherder.sh
+
+LOGFILE=/var/log/manage/ingest_from_pulse.log
```

Comment:
The manage directory won't exist, so the `touch` will fail, unless we get fubar to create manually on whatever nodes this end up running on. Perhaps the log will be fine as `/var/log/treeherder_ingest_from_pulse.log`? (Post Heroku move this will all go away anyway :-))